ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/SLES.yml

```diff
@@ -45,6 +63,7 @@ Additional_Build_Tools_SLES11:
 Additional_Build_Tools_SLES_x86:
   - glibc-32bit                   # a dependency required for executing a 32-bit C binary
   - glibc-devel-32bit             # a dependency required for executing a 32-bit C binary
+  - libelf0
```

Comment:
Ran the playbook on a x86_64 SLES 15 SP0 fyre machine. This package failed to install, with an error message along the lines of No provider of libelf0 found.

Reply:
AH! Sorry, I must've put it in the wrong place.